### PR TITLE
Layout/spacing fixes

### PR DESCRIPTION
**Pull request by ashenden**
* Updated CNVS to 1.1.4
* Removed spacing between options gear icon and caret
* Increased height of `.page-body-content` margins
* Removed margin classes from `.configuration-map-heading`s
* Adjusted `.container` widths

### CODE REVIEW

File: src/styles/layout/sidebar/variables.less [hunk 1]
```diff
@@ -15,6 +15,14 @@
 @sidebar-screen-large-enabled: @layout-screen-large-enabled;
 @sidebar-screen-jumbo-enabled: @layout-screen-jumbo-enabled;
 
+/* Layout */
+
+// Width
+
+@sidebar-width-screen-medium: 180px;
+@sidebar-width-screen-large: 200px;
+@sidebar-width-screen-jumbo: 240px;
```
Comment (jfurrow) [inline src/styles/layout/sidebar/variables.less]: These variables are already defined on lines 7 through 11 in this file

**Comment by jfurrow** [comment]
lol @ hack job, not sure what you're referring to though as the variables are still there :)

File: src/styles/layout/sidebar/variables.less (hunk 1 above)
Comment (ashenden) [inline src/styles/layout/sidebar/variables.less]: Hmm.  Must have overlooked them.  Things should be good now though.

**Comment by ashenden** [comment]
"Hack job" = removing all my vars and mixins :(

**Comment by jfurrow** [comment]
@ashenden oh, that...

I updated the shrinkwrap in this PR also.

**Comment by ashenden** [comment]
Ready?

**Comment by jfurrow** [comment]
@ashenden Let's wait until we get two labels of approval.

**Comment by orlandohohmeier** [comment]
LGTM! 🚢

**Pull request merged**